### PR TITLE
Changes the base amount of bone gel.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
Changed the amount from 1 to 5 for original `bone_gel` so it's easier for mappers to add the right bone gel.
Deleted subtype `/four` and added subtype `/one` specifically for bone gel creating.
Increased from 4 to 5 because surgical tape has 5.
## Why It's Good For The Game
Having 1 bone gel near 5 surgical tapes always seemed off.
## Changelog
:cl:
balance: Bone gel standart amount has been increased to 5.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
This should be qol not balance

**Comment by Helg2** [comment]
> This should be qol not balance

Its subjective

**Comment by ExcessiveUseOfCobblestone** [comment]
epic

**Comment by TheVekter** [comment]
standart

**Comment by necromanceranne** [comment]
You shouldn't be losing gbp for fixing what feels like more of a design oversight.

**Comment by CoiledLamb** [comment]
real shit!! save your gbp op, there's still time!!

File: _maps/RandomRuins/SpaceRuins/infested_frigate.dmm [hunk 1]
```diff
@@ -1109,7 +1109,7 @@
 	},
 /obj/machinery/vending/medical{
 	req_access = list("theatre");
-	products = list(/obj/item/stack/medical/gauze=0,/obj/item/reagent_containers/syringe=7,/obj/item/reagent_containers/dropper=3,/obj/item/healthanalyzer=0,/obj/item/wrench/medical=0,/obj/item/stack/sticky_tape/surgical=0,/obj/item/healthanalyzer/wound=0,/obj/item/stack/medical/ointment=0,/obj/item/stack/medical/suture=1,/obj/item/stack/medical/bone_gel/four=1,/obj/item/cane/white=2)
+	products = list(/obj/item/stack/medical/gauze=0,/obj/item/reagent_containers/syringe=7,/obj/item/reagent_containers/dropper=3,/obj/item/healthanalyzer=0,/obj/item/wrench/medical=0,/obj/item/stack/sticky_tape/surgical=0,/obj/item/healthanalyzer/wound=0,/obj/item/stack/medical/ointment=0,/obj/item/stack/medical/suture=1,/obj/item/stack/medical/bone_gel=1,/obj/item/cane/white=2)
```
Comment (san7890) [inline _maps/RandomRuins/SpaceRuins/infested_frigate.dmm:1112]: god damn this is ugly

Comment (san7890) [reply _maps/RandomRuins/SpaceRuins/infested_frigate.dmm:1112]: you definitely don't need to change it in this PR but why the hell isn't this subtyped i throw up

Comment (Helg2) [reply _maps/RandomRuins/SpaceRuins/infested_frigate.dmm:1112]: I don't mind

**Pull request merged**